v4l2-request-test.c

```diff
@@ -306,6 +338,10 @@ int main(int argc, char *argv[])
 		case '?':
 			print_help();
 			goto error;
+		default:
+			printf("Unknown option (0%o)\n",
+			       opt);
```

Comment:
I think we can just ignore unknown options since we already print out a summary.

Reply:
ok. will throw it away.